v2/util/docker-cleanup.sh

```diff
@@ -1,17 +1,27 @@
 #!/bin/bash
 
-# this exists because trying to escape bash commands in a sytemd unit is super sadtimes
+# NOTE: this script will need to be sudo'ed for access to /var/lib/docker
```

Comment:
I thought we were standardizing as `core` and only elevating, explicitly, when needed.